testing/trino-testing-containers/src/main/java/io/trino/testing/containers/TestContainers.java

```diff
@@ -14,6 +14,10 @@
 
```

Comment:
> It would make more sense to limit the change to PostgreSqlQueryRunner.main

I've another use case: 
I breakpoint a test which I'm interested, because of setup which takes place in that test class. And within another run I try connect to the environment which was created during the first run. 
I do it in very straight-forward way - mostly I'm not interested in the specific setup of the environment. I just want to start/rerun a few times a test. With limiting the scope of `Expose postgresql in container on default port` to the `main` method will force me to do this setup manually, and by Murphy's law I'll do it wrong.

Can we introduce some new toggle to enable/disable this feature for all methods in the QueryRunner?

Reply:
Ideally yes. 
As discussed with @vlad-lyutenko 
```
public TestingPostgreSqlServer(boolean shouldExposeFixedPort)
``` 
Provides good enough extension point. So during container startup I can hardcode `true` as a parameter. 
So please disregard my comment above.